utils/polus-python-template/{{cookiecutter.container_name}}/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "{{cookiecutter.project_name}}"
+version = "{{cookiecutter.plugin_version}}"
+description = "{{cookiecutter.plugin_description}}"
+authors = ["{{cookiecutter.author}} <{{cookiecutter.author_email}}>"]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.9.15"
+typer = "^0.7.0"
+bfio = {version = "2.1.9", extras = ["all"]}
+filepattern = "2.0.0"
+preadator = "0.3.0.dev1"
+
+[[tool.poetry.source]]
+name = "test"
+url = "https://test.pypi.org/simple/"
+default = false
+secondary = true
+
+[tool.poetry.group.dev.dependencies]
+bump2version = "^1.0.1"
+flake8 = "^6.0.0"
+pre-commit = "^3.2.1"
+flake8-docstrings = "^1.7.0"
+black = "^23.3.0"
+mypy = "^1.1.1"
+pytest = "^7.2.2"
+ruff = "^0.0.270"
```

Comment:
Remove `flake8` and `flake8-docstrings` because that job is now handled by `ruff`.

Also, these versions are like to be updated every time we make a new plugin so perhaps they should have looser dependencies or should not be listed here at all?